frontend/src/pages/agency-profile-form/AgencyProfileForm.js

```diff
@@ -789,7 +885,7 @@ class AgencyProfileForm extends Component {
                 className="form-button-submit"
                 onClick={this.submitForm}
               >
-                {this.props.edit ? "Save Profile" : "Create Profile"}
+                {this.props.editSection ? "Save Profile" : "Create Profile"}
```

Comment:
what is editSection being derived from? I'm assuming this is for future editing functionality and isn't being used right now? Correct me if I'm wrong

Reply:
Yep, editSection will (most likely) be passed as a prop from the page that displays all the info